docs/source/index.rst

```diff
@@ -167,6 +167,7 @@ Whitepapers
    api/nucypher.network
    api/nucypher.datastore
    api/nucypher.crypto
+   api/nucypher.acumen
```

Comment:
Yeah, I know it's a bit of a strange name.  We've gone around and around about it.  I have an increasingly vivid imagination of spinning acumen out into its own node discovery and instantiation library, which I think will be useful for other projects.  So I like the idea of choosing something that also makes sense as a library.

Reply:
This package is currently(?) limited to node nicknames and fleet states. 

Perhaps thinking about it as a standalone library is a good way to gain perspective. `nicknames.py` and `perception.py` seem somewhat disjointed to me. 
I like the use of the term `perception` perhaps that could be made the package name instead of `acumen`? The current`perception.py` file changed to `fleet.py` and `nicknames.py` to `node.py` - that way we have `node.py` and `fleet.py` modules within `perception`?

From previous discussions alternative names were:
- comprehension
- knowledge
- <https://discordapp.com/channels/411401661714792449/411401661714792451/713528057071403080>